libs/components/text-editor/src/lib/modules/text-editor/toolbar/text-editor-toolbar.component.ts

```diff
@@ -158,7 +162,10 @@ export class SkyTextEditorToolbarComponent implements OnInit {
     const inputModal = this.#modalService.open(SkyTextEditorUrlModalComponent, [
       {
         provide: SkyUrlModalContext,
-        useValue: { urlResult: currentLink },
+        useValue: {
```

Comment:
You didn't write this initially, but the usage of an object literal where a class instance is expected is an anti-pattern. Would you mind updating this code to use the class instead of an object literal?

```
provide: SkyUrlModalContext,
useFactory: () => {
  const context = new SkyUrlModalContext();
  context.urlResult = currentLink;
  context.linkWindowOptions = this.linkWindowOptions;

  return context;
}
```

Reply:
Done!